fourohfour/Dockerfile

```
@@ -1,21 +1,13 @@
 # arxiv/fourohfour
 
-FROM arxiv/base:0.14.2
+FROM arxiv/base:ARXIVNG-2462
```

How about parametrizing it like this?

```suggestion
ARG     ARXIVNG_VER=ARXIVNG-2462
FROM arxiv/base:${ARXIVNG_VER}
```

The default value can be something more generic such as `latest`, but in the build pipeline proper `--build-arg` can be supplied to externally supply the value without the need of updating this Docker file each time a new base image is ready.

---

Nice, I like that

---

Added this in dce0639